List.tsx

```diff
@@ -0,0 +1,28 @@
+import { useSelector } from "react-redux";
+import { IState } from "./store";
+import React from "react";
+import "./style.css";
+
+export const List: React.FC = () => {
+  const listOfUsers = useSelector((state: IState) => {
+    console.log(state);

```

Comment:
No logs
It's a good practice to create a custom selector, so you xan reuse them in future 

And why you didn't reuse useUserSelector?

Reply:
rewrote useSelector several times, because it didn't work,
I made all the changes in the List file, so I forgot to return to the custom selector.